galsim/gsobject.py

```diff
@@ -1604,14 +1638,15 @@ def drawImage(self, image=None, nx=None, ny=None, bounds=None, scale=None, wcs=N
         # Figure out what wcs we are going to use.
         wcs = self._determine_wcs(scale, wcs, image)
 
-        # Make sure offset is a PositionD
+        # Make sure offset and center are PositionD if entered as tuples.
```

Comment:
I think these convenience functions also work if offset/center are entered as lists or a numpy array?  Maybe this documentation could be something like "Make sure offset and center are PositionD, converting as necessary from other formats.  If `center` was passed in as None, it will remain that way after `_parse_center` is run"?